simulation/fake_peer.hpp

```diff
@@ -180,6 +172,17 @@ struct fake_peer
 			});
 	}
 
+private:
+
+	void send_simple(std::uint8_t msg)
```

Comment:
may I suggest a different name? like `send_simple_message` or `send_small_message`? also is `msg` a "message code"?

Reply:
yes, makes sense